packages/components/table/src/table-column/index.ts

```diff
@@ -67,9 +67,13 @@ export default defineComponent({
 
       const type = props.type || 'default'
       const sortable = props.sortable === '' ? true : props.sortable
-      const showOverflowTooltip = isUndefined(props.showOverflowTooltip)
-        ? parent.props.showOverflowTooltip
-        : props.showOverflowTooltip
+      //Restrict the `selection` type to set `showOverflowTooltip`, as its own label tag cannot use the text-align property of the parent element. Therefore, it is directly restricted.
```

Comment:
*The selection column should not be affected by `showOverflowTooltip`.*

Perhaps a brief description will suffice.

Reply:
Modified